### PR TITLE
chore: RefreshContainer

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by MartinZikmund** [comment]
Please ensure the code is updated to the latest sources, I think there might have been changes since

**Comment by Youssef1313** [comment]
Note to self: Not implemented APIs used by RefreshContainer:

```
CreateScalarKeyFrameAnimation (can be worked around by moving to the final value without animating)
CreateVector2KeyFrameAnimation (can be worked around by moving to the final value without animating)
CancelDirectManipulations (maybe not needed)
ScrollViewer.ViewChanging (?)
ScrollViewer.DirectManipulationCompleted (?)
PositionXSourceMode (?)
PositionYSourceMode (?)
PositionXChainingMode (?)
PositionYChainingMode (?)
```

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/RefreshVisualizer/RefreshVisualizer.cs [hunk 1]
```diff
@@ -477,10 +389,11 @@ private void ExecuteInteractingAnimations()
 			string interactionRatioPropertyName = m_refreshInfoProvider.InteractionRatioCompositionProperty;
 			CompositionPropertySet interactionRatioPropertySet = m_refreshInfoProvider.CompositionProperties;
 
+			// Uno specific: "f" suffixes are removed as they are not supported.
```
Comment (Youssef1313) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/RefreshVisualizer/RefreshVisualizer.cs:392]: Note to self: They are now supported and the workaround should be removed

**Review by MartinZikmund — APPROVED** [review]

**Pull request merged**